_assets/img/Integrate-Web-Search.svg

```diff
@@ -0,0 +1 @@
"#7542e5"/><stop offset="0.42" stop-color="#824deb"/><stop offset="0.79" stop-color="#a067fa"/><stop offset="1" stop-color="#ab71ff"/></linearGradient><clipPath id="clip-path"><rect x="-17.67" y="519.08" width="1240" height="506.67" style="fill:url(#linear-gradient)"/></clipPath><linearGradient id="linear-gradient-8" x1="1748.59" y1="-15.34" x2="-94.3" y2="1363.38" gradientUnits="userSpaceOnUse"><stop offset="0.2" stop-color="#b833e1"/><stop offset="0.75" stop-color="#592acb"/></linearGradient></defs><title>amo_dev-hub_illustration-assets-v2-EDITED</title><path d="M166.88,155.23a22.46,22.46,0,0,1-22.36,22.46H54.62a17.17,17.17,0,0,1,0-34.33,17,17,0,0,1,4.12.52A17.7,17.7,0,0,1,86,125.3a29.75,29.75,0,0,1,58.54,7.48A22.45,22.45,0,0,1,166.88,155.23Z" style="fill:url(#radial-gradient)"/><circle cx="263.52" cy="197.28" r="162.45" style="fill:url(#radial-gradient-2)"/><path d="M463.76,93.7a25.7,25.7,0,0,0-6.18.75,26.3,26.3,0,0,0,.69-6,26.71,26.71,0,0,0-41.77-22,44.85,44.85,0,0,0-88.23,11.28,33.69,33.69,0,1,0,0,67.38H463.76a25.72,25.72,0,1,0,0-51.43Z" style="fill:url(#radial-gradient-3)"/><rect x="54.69" y="127.01" width="297.55" height="48.25" rx="5.4" style="fill:url(#linear-gradient-2)"/><path d="M98.59,163.7,88.24,153.36a12.91,12.91,0,1,0-3,3l10.34,10.34a2.15,2.15,0,0,0,3-3Zm-20.83-9.21a8.58,8.58,0,1,1,8.58-8.58A8.58,8.58,0,0,1,77.76,154.49Z" style="fill:#f9f9fa"/><path d="M348,85H166a13.65,13.65,0,0,0-13.65,13.65V229.54A13.65,13.65,0,0,0,166,243.19H352.57a9.1,9.1,0,0,0,9.1-9.1V98.65A13.65,13.65,0,0,0,348,85Z" style="fill:#0df"/><path d="M347.31,85h-182a13.65,13.65,0,0,0-13.64,13.65V229.54a13.65,13.65,0,0,0,13.64,13.65H351.86a9.1,9.1,0,0,0,9.09-9.1V98.65A13.65,13.65,0,0,0,347.31,85Zm4.55,18.2H160.77V98.65a4.55,4.55,0,0,1,4.55-4.55h182a4.55,4.55,0,0,1,4.55,4.55Z" style="fill:url(#linear-gradient-3)"/><rect x="168.79" y="152.24" width="297.55" height="48.25" rx="5.4" style="fill:url(#linear-gradient-4)"/><path d="M212.69,188.94l-10.34-10.35a12.94,12.94,0,1,0-3,3L209.66,192a2.14,2.14,0,0,0,3-3Zm-20.82-9.21a8.58,8.58,0,1,1,8.58-8.58A8.58,8.58,0,0,1,191.87,179.73Z" style="fill:#f9f9fa"/><rect x="102.01" y="261.86" width="297.55" height="48.25" rx="5.4" style="fill:url(#linear-gradient-5)"/><path d="M145.9,298.55l-10.34-10.34a12.91,12.91,0,1,0-3,3l10.34,10.35a2.15,2.15,0,0,0,3-3Zm-20.82-9.21a8.58,8.58,0,1,1,8.58-8.58A8.58,8.58,0,0,1,125.08,289.34Z" style="fill:#f9f9fa"/><path d="M413.26,342.11l-38.48-38.47a48,48,0,1,0-11.28,11.28L402,353.39a8,8,0,0,0,11.28-11.28Zm-77.45-34.25a31.92,31.92,0,1,1,31.91-31.92A31.92,31.92,0,0,1,335.81,307.86Z" style="fill:url(#linear-gradient-6)"/><rect x="-17.67" y="519.08" width="1240" height="506.67" style="fill:url(#linear-gradient)"/><g style="clip-path:url(#clip-path)"><path d="M1501.3,53.94a170.95,170.95,0,0,0-311.48,141c127.79,282.34,2.07,616-280.27,743.79s-616,2.06-743.78-280.28a171,171,0,0,0-311.48,141c205.53,454.07,742.17,656.29,1196.24,450.76S1706.82,508,1501.3,53.94Z" style="fill:url(#linear-gradient-8)"/></g></svg>
```

Comment:
Note: Style via attributes won't be allowed by the current CSP.

Reply:
I've re-exported the images too use internal stylesheets.

Reply:
Would you be able export them with pretty printed markup? The build should optimize them so the source doesn't need to be pre-optimized and that way it's easier to see the content without horizontal scrolling.

Reply:
Unfortunately style blocks inside the SVG will also be forbidden by same rules in the current CSP. Can you use an external CSS file instead?

Reply:
@muffinresearch I'm not sure how to automate the process of pretty markup for the svgs. They are graphics generated by Adobe Illustrator. I also don't think it's feasible to make the styles external. These are images that could be replaced at any time in the future and I think they should be completely encapsulated. With the exception of the Anatomy of an Extension graphics, the svgs are being included on the page as images or background images... do the same blocking rules apply to images as they would to svg code that is embedded on the page?

Reply:
How are you exporting from illustrator? Can you use "Export As" and set:

* Styling to "presentation attributes".
* Uncheck minify

Hopefully that will make a difference.

Reply:
Oh cool! Ok, testing that out.

Reply:
That looks good. Thanks 👍